sdk/src/budget_transaction.rs

```diff
@@ -81,13 +81,17 @@ impl BudgetTransaction {
     pub fn new_signature(
         from_keypair: &Keypair,
         contract: Pubkey,
-        to: Pubkey,
+        to: Option<Pubkey>,
```

Comment:
Why change this interface?  Instead, change that new conditional to: `key != to`?

Reply:
Haha, I wrote it that way first, but thought it sort of obscured what was happening.
Looks all right by the light of day, though. Will change back.